src/extension.ts

```diff
@@ -646,7 +788,6 @@ export function activate(context: vscode.ExtensionContext) {
           placeHolder: "Select AI provider",
           title: "Configure API Type",
           canPickMany: false,
-          ignoreFocusOut: true,
```

Comment:
this matches how most other default status bar clicks handle popups/dropdowns, felt smoother